src/katsdpcalproc/pointing.py

```diff
@@ -236,23 +233,37 @@ def __call__(self, x):
         """
         return self._interp(x)
 
+# Helper function to calculate expected width
+
+
+def _get_widths(beamwidth, freqs, diameter):
+    expected_width = rad2deg(beamwidth * lightspeed
+                             / freqs / diameter)
+    expected_width = np.sqrt(2.0) * expected_width
+    # XXX This assumes we are still using default ant.beamwidth of 1.22
+    # and also handles larger effective dish diameter in H direction
+    expected_width = (0.8 * expected_width, 0.9 * expected_width)
+    return expected_width
+
 
-def beam_fit(data_points, NUM_CHUNKS, ants):
+def beam_fit(data_points, ants, num_chunks=16):
     """Fit primary beams to receptor gains obtained at various offset pointings.
 
     Parameters
     ----------
 
-    data_points : dict mapping receptor index to (x, y, freq, gain, weight) seq
-        Complex gains per receptor, as multiple records per offset and frequency
-    NUM_CHUNKS: Group the frequency channels into this many sections to obtain
+    data_points : :class: 'dict'
+        dict mapping receptor index to (x, y, freq, gain, weight) seq
+        Complex gains per receptor, as multiple records per offset and
+        frequency chunk
+    ants: list of :class: 'katpoint.Antenna'
+        A list of antenna objects for fitted beams
+    num_chunks: :class: 'float'
```

Comment:
For plain old types, you can just say dict or float. Take note that custom classes use backticks (`) and not quotes ('), without a space between :class: and the rest.

For more details of the docstring convention, read the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) docs (and then if you need it, the [Sphinx](https://www.sphinx-doc.org/en/master/usage/referencing.html) docs). And check the existing / original code, which should be compliant with this style.

If you are unsure of something, look at the lines of code around yours and follow their lead (good general tip!).